lsm-srlinux/ci/Jenkinsfile

```
@@ -157,11 +157,17 @@ pipeline {
                             python${python_version} -m venv venv
                             source "venv/bin/activate"
                             pip install -U pip
+
+                            # Get inmanta version from the container
+                            inmanta_version=$(docker inspect --format='{{index .Config.Labels "com.inmanta.version"}}' clab-srlinux-inmanta-server)
+                            echo "inm_version=$inmanta_version"
```

Why abbreviate debugging output:
```suggestion
                            echo "inmanta_version=$inmanta_version"
```